src/panels/lovelace/elements/hui-state-label-element.ts

```diff
@@ -58,6 +59,25 @@ class HuiStateLabelElement extends LitElement implements LovelaceElement {
       `;
     }
 
+    if (attribute && !stateObj.attributes[attribute]) {
+      return html`
+        <hui-warning-element
+          label=${this.hass.localize(
+            "ui.panel.lovelace.warning.attribute_not_found",
+            "attribute",
+            this._config.attribute,
+            "entity",
+            this._config.entity
+          )}
+        ></hui-warning-element>
+      `;
+    }
+
+    const stateDisplay = stateObj
+      ? computeStateDisplay(this.hass.localize, stateObj, this.hass.language)
+      : "-";
+    const attributeDisplay = stateObj ? stateObj.attributes[attribute] : "-";
```

Comment:
remove this. stateObj exists

Reply:
I see why you did check as it was in the existing code, but that was unnecessary